### DISCUSSION TITLE
SCPH-5501 Slot 2 Problem in Rayman (USA)

### DISCUSSION THREAD

**Discussion by kittehcode** [brad-lin/FreePSXBoot#73] [General]
I have a SCPH-5501. Using Tomee memory cards. I can use the exploit on slot 1, and it works great.

On slot 2 for some titles, however Rayman (USA) for example crashes (freezes) when trying to load, save, or create a game on memory card one. This holds true even if I remove the freepsxboot from slot 2 before starting the game. All other games I tested worked fine.

**Comment by brad-lin** [comment]
Could you try with this image and let me know? Thanks

https://github.com/brad-lin/FreePSXBoot/files/8306888/freepsxboot-unirom-fastload-20220318-bios-3.0-1996-11-18-A-8d8cb7e4-slot2.zip

**Comment by JMaxxL** [comment]
I tried rayman @ 8x using real sony memory card slotwo w/ larger PCB & had better luck with v1.4.5 on my SCPH-7501 via vertbatium
